bokehjs/src/lib/models/text/math_text.ts

```diff
@@ -414,7 +412,7 @@ export class TeXView extends MathTextView {
 
   protected _process_text(text: string): HTMLElement | undefined {
     // TODO: allow plot/document level configuration of macros
-    return this.provider.MathJax?.tex2svg(text, this.model.macros)
+    return this.provider.MathJax?.tex2svg(text, undefined, this.model.macros)
```

Comment:
I actually can't find ref docs for `MathJax.tex2svg` despite searching. What is the undefined for (can you give a pointer to the docs)

Reply:
This should not be here. This second options are the [conversion options](http://docs.mathjax.org/en/latest/web/typeset.html?highlight=display%3A#conversion-options) documented on MathJax/index.ts too